applications/serializers.py

```diff
@@ -94,3 +98,46 @@ class BootcampApplicationSerializer(serializers.ModelSerializer):
     class Meta:
         model = models.BootcampApplication
         fields = ["id", "state", "created_on", "bootcamp_run"]
+
+
+class SubmissionReviewSerializer(SubmissionSerializer):
+    """ApplicationStepSubmission serializer for reviewers"""
+
+    bootcamp_application = BootcampApplicationSerializer(required=False, read_only=True)
+    learner = UserSerializer(
+        source="bootcamp_application.user", required=False, read_only=True
+    )
```

Comment:
Including the entire Learner serializer here maybe returns more data for the submissions list than necessary, but will save an additional API call on the submission detail UI to get that info.  With paging set to 10 the response size shouldn't be an issue though.

Reply:
yeah, I originally tried to pull in https://django-rest-framework-serializer-extensions.readthedocs.io/ to make this cleaner, but that ended being a tug of war between having it work for both reads here and writes elsewhere in the app. Decided to take the performance hit for the sake of not spending more time on it... :slightly_frowning_face: